en/embedding.html

```diff
@@ -41,7 +41,11 @@ <h2 id="provided-embedders">Provided embedders</h2>
 <h3 id="bertbase-embedder">BertBase embedder</h3>
 
 <p>An embedder using <a href="#wordpiece-embedder">WordPiece</a> to produce tokens which is then input to
-a supplied ONNX model on the form expected by a BERT base model.</p>
+a supplied ONNX model on the form expected by a BERT base model. 
+See <a href="https://github.com/vespa-engine/sample-apps/blob/master/simple-semantic-search/export_model_from_hf.py">export_model_from_hf.py</a>,
+for how to export a <a href="https://www.sbert.net/">sentence-transformer</a> model to ONNX format.
+See also <a href="#troubleshooting-model-signature">troubleshooting model signature</a>.
+</p>
```

Comment:
https://github.com/vespa-engine/sample-apps/tree/master/examples/model-deployment might also be useful